ga4gh/datamodel/variants.py

```diff
@@ -269,6 +269,7 @@ class HtslibVariantSet(datamodel.PysamDatamodelMixin, AbstractVariantSet):
     """
     def __init__(self, id_, dataDir):
         super(HtslibVariantSet, self).__init__(id_)
+        self._dataDir = dataDir
```

Comment:
Sure, but to where?  `AbstractVariantSet` would not eliminate the redundancy, and `DatamodelObject` is too general.  We could do something in `PysamDatamodelMixin`, but that doesn't have any `__init__` method...